addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
-        <xpath expr="//*[hasclass('o_Discuss_thread')]" position="before">
-            <t t-if="discuss.thread.visitor">
+    <t t-inherit="mail.ThreadView" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_ThreadView_threadViewTopbar')]" position="after">
```

Comment:
If the thread view doesn't show topbar, it doesn't show Visitor Banner. I don't think that is what we want. The presence of Visitor Banner should not be coupled with presence of topbar.